window_manager/options.js

```diff
@@ -45,13 +45,84 @@ function validateOptions() {
     valid = false;
   }
 
-  if (valid) {
-    setStatus('Valid');
+  // JSON validation completed
+  //
+  if (! valid) {
+    setStatus('Invalid JSON');
+    return valid;
   }
 
+  const hasWarnings = ! await warnForIncorrectMonitorIds(actionsObj);
```

Comment:
yes it just shows a warning, and does not block the save. 

I added this because I miss-spelled a monitor name and was cursing trying to find out why the action didn't work!